src/index.js

```diff
@@ -615,11 +624,17 @@ export function useDropzone(options = {}) {
           files.forEach(file => {
             const [accepted, acceptError] = fileAccepted(file, accept)
             const [sizeMatch, sizeError] = fileMatchSize(file, minSize, maxSize)
-            if (accepted && sizeMatch) {
+            const customErrors = validator ? validator(file) : null;
```

Comment:
Might make sense to check if `validator` is a function?

Reply:
I don't think it makes sense. There is TypeProps specified for this property and it should warn about incorrect values.  Also ignoring non-function values is more confusing, than getting "TypeError: ___ is not a function'. Adding such check makes sense only to throw error with better message.

Reply:
I'm not saying to ignore it, I'm saying to maybe throw a more friendly message, but I guess it's ok to leave as is. As for TypeProps, do note that if ppl use this as a hook, they would get no warnings.